src/modules/subcommands/handleBreakout.ts

```diff
@@ -40,17 +51,33 @@ export const handleBreakout: CommandHandler = async (bot, interaction) => {
       const targetMember = getRandomValue(mutableMembers);
       const index = mutableMembers.findIndex((m) => m.id === targetMember.id);
       mutableMembers.splice(index, 1);
-      await targetMember.voice.setChannel(newChannel);
+      try {
+        await targetMember.voice.setChannel(newChannel);
+      } catch (e) {
+        console.log(e);
+      }
     }
   }
 
+  try {
+    mutableMembers.forEach(
+      async (m) => await m.voice.setChannel(newChannels[0])
+    );
+  } catch (e) {
+    console.log(e);
+  }
```

Comment:
```suggestion
  mutableMembers.forEach(async (m) => {
    try {
      await m.voice.setChannel(newChannels[0]);
    } catch (e) {
      console.log(e);
    }
  });
```

@eddiejaoude I'm assuming this branch is deployed...

The issue with #22 is because the error comes from within the `.forEach` callback which isn't passed up to the try/catch, I believe.

Reply:
Ah good spot 👍